packages/vscode-extension/src/handlers/copilotChatHandlers.ts

```diff
@@ -64,20 +60,24 @@ async function openGithubCopilotChat(query: string): Promise<Result<null, FxErro
 
 export async function installGithubCopilotChatExtension(
   triggerFrom: TelemetryTriggerFrom
-): Promise<Result<{ installCopilot: boolean }, FxError>> {
-  const eventName = "installCopilotChat";
+): Promise<Result<null, FxError>> {
+  const startEventName = "install-copilot-chat-start";
```

Comment:
Better move telemetry event names to existing extTelemetryEvents.ts file. You can also update in next PR.